pyproject.toml

```
@@ -71,6 +71,14 @@ lint.extend-select = [
 lint.isort = {known-first-party = ["helpers", "package_info", "pipx"]}
 lint.mccabe.max-complexity = 15
 
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git,*.pdf,*.svg,go.sum'
```

```suggestion
skip = '.git,*.pdf,*.svg'
```
I don't think we will have a `go.sum` in this project.

---

will remove then